src/select/stories/constants.js

```diff
@@ -65,7 +66,27 @@ const stateOptions = [
   { value: 'WY', label: 'Wyoming' },
 ];
 
+const generateOptions = (count :number) :Object[] => {
+
+  const options = [];
+  const chars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+  for (let i = 0; i < count; i += 1) {
+    let label = '';
+    let value = '';
+    for (let j = 0; j < 10; j += 1) {
+      label += chars[Math.floor(Math.random() * chars.length)];
+    }
+    value = chars[Math.floor(Math.random() * chars.length)];
```

Comment:
This is always 1 letter. Is this intentional?

Reply:
oops thanks for catching this! it was intentional, but i was just temporarily comparing with have a smaller value vs. a longer one.